src/app/zap-templates/zcl/wake-on-lan-cluster.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+  <cluster>
+    <domain>General</domain>
+    <name>Wake on LAN</name>
+    <code>0x0503</code>
+    <define>WAKE_ON_LAN_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <description>This cluster provides an interface for managing low power mode on a device that supports the Wake On LAN protocol.</description>
+    <attribute side="server" code="0x0000" define="WAKE_ON_LAN_MAC_ADDRESS" type="OCTET_STRING" length="32" writable="false" optional="false">wake on lan mac address</attribute>
```

Comment:
attribute in spec seems to be string rather than binary string. Is that supported for attributes?

Reply:
Updated to CHAR_STRING.